horovod/common/parameter_manager.h

```diff
@@ -46,13 +49,14 @@ class ParameterManager {
   ParameterManager();
 
   // Creates MPI data types used for sending and receiving optimization state across workers.
-  void CreateMpiTypes();
+  void static CreateMpiTypes(MPI_Datatype &mpi_params_type_);
```

Comment:
Let's remove these Create and Free methods.

Reply:
Removed.